src/cms/forms/pages/page_translation_form.py

```diff
@@ -78,6 +78,8 @@ def save(self, commit=True):
         if not {"slug", "title", "text"}.isdisjoint(self.changed_data):
             self.instance.version += 1
             self.instance.pk = None
+        else:
+            self.instance.status = status.PUBLIC
```

Comment:
Hmm, I'm not sure if this is the right way to go. This will set the status to public each time a new page is created, but sometimes, the users want to create a page as draft...
I think we need to introduce a special `AUTOSAVE` status to the [constants](https://github.com/Integreat/integreat-cms/blob/develop/src/cms/constants/status.py) which would also be helpful for people to decide whether they want to stick with the autosaved version or restore the last "normal" version...

Reply:
When some changes are really made (not afterwards deleted before the auto save), it creates a new version as draft and the original stays public.

Reply:
Oh right, sorry! But still there is the case when you make no changes but click on the button "Save as draft", it makes the status public although it should be draft...

Reply:
I introduced, as you suggested, a new status AUTO for auto save and updated the behaviour of auto save.
Now the user can save the page as draft by cliking "save as draft" also when no real change has been made.